evalml/automl/automl_search.py

```diff
@@ -1714,6 +1714,30 @@ def full_rankings(self):
         rankings_df.reset_index(drop=True, inplace=True)
         return rankings_df
 
+    def get_recommendation_score_breakdown(self, pipeline_id):
```

Comment:
where do we display the weights to the user? I don't see anything [here](https://evalml.alteryx.com/en/latest/user_guide/automl.html#Recommendation-Score) so far. If the purpose of `get_recommendation_score_breakdown` is to explain to the user how the recommendation score is calculated, I think we should give them the weights so they know how they go from the metrics to the recommendation score.

Reply:
The weights aren't directly displayed to the user, but the docstring states that they are weighted equally by default and we offer methods for customization by the user. 

The goal of this function isn't to explicitly explain to the user how the recommendation score was calculated - that's why the values reported are explicitly the non-normalized ones. Rather, the goal is to present the user with the evaluated objectives behind the recommendation score. What they do with that information is up to them, but it's more to give them the insight about direct model performance rather than "here's the equation".

Reply:
works for me!